src/controllers/items/getItems.ts

```diff
@@ -11,7 +12,14 @@ export default [
   // Controller
   async (request: Request, response: Response, next: NextFunction) => {
     try {
-      const items = await fetchItems();
+      const { team } = getRequestInfo(response);
+      let items = await fetchItems();
+
+      // Check if user is not in SSBU tournament
+      if (!team || team.tournamentId == null || team.tournamentId !== 'ssbu') {
```

Comment:
```suggestion
      if (!team || team.tournamentId !== 'ssbu') {
```
T'es pas obligé de vérifier que le tournament est null si tu le compares après avec SSBU